lib/tether/ops_linux.go

```diff
@@ -699,6 +700,56 @@ func (t *BaseOperations) MountTarget(ctx context.Context, source url.URL, target
 	return nil
 }
 
+// CopyExistingContent copies the underlying files shadowed by a mount on a directory
+// to the volume mounted on the directory
+// this is only done only once
+// see bug https://github.com/vmware/vic/issues/3482
+func (t *BaseOperations) CopyExistingContent(source string) error {
+	defer trace.End(trace.Begin(fmt.Sprintf("copyExistingContent from %s", source)))
+
+	source = filepath.Clean(source)
+
+	log.Debugf("creating directory %s", bindDir)
+	if err := os.MkdirAll(bindDir, 0644); err != nil {
+		log.Errorf("error creating directory %s: %+v", bindDir, err)
+		return err
+	}
+
+	parentDir := filepath.Dir(filepath.Clean(source))
```

Comment:
Sure thing 😄 yeah I was mainly just looking at the filepath.clean in the function call.